lib/matplotlib/legend.py

```diff
@@ -196,6 +196,12 @@ def _update_bbox_to_anchor(self, loc_in_canvas):
     The legend's background patch edge color.
     If ``"inherit"``, use :rc:`axes.edgecolor`.
 
+linewidth : float or None, default: :rc:`legend.linewidth`
+    The legend's background patch edge linewidth.
+    If ``None``, use :rc:`axes.linewidth`.
+
+    .. versionadded:: 3.10
```

Comment:
```suggestion
    .. versionadded:: 3.11
```
v3.10 is already released.